### DISCUSSION TITLE
请问用FIL储存数据价格大概多少？

### DISCUSSION THREAD

**Discussion by hinego** [filecoin-project/community-china#209] [新手上路]
储存1T数据一年大概要多少币？或者多少钱

哪里可以看到，怎么计算

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
